packages/@aws-cdk/aws-stepfunctions-tasks/lib/athena/get-query-execution.ts

```diff
@@ -0,0 +1,62 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Construct } from 'constructs';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * Properties for getting a Query Execution
+ * @experimental
+ */
+export interface AthenaGetQueryExecutionProps extends sfn.TaskStateBaseProps {
+  /**
+   * Query that will be retrieved
```

Comment:
add an example query execution id if possible - it's helpful for customers to see `i.e. adfsaf-23trf23-f23rt23` to get a sense of what it represents in addition to the description provided in the doc string.

Reply:
sorry I was unclear... this was just an example - a real one (grabbed from your test looks like `2da557a1-7283-4c3d-8af9-058348f0bb02`)